src/main/aws/sso.ts

```
@@ -0,0 +1,147 @@
+      return stored;
+    }
+
+    const response = await this.ssoOidcClient.send(
+      new RegisterClientCommand({
+        clientName: "DynoView",
+        clientType: "public",
+      }),
+    );
+
+    const registration = {
+      clientId: response.clientId!,
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)